src/foundation.cs

```diff
@@ -8058,6 +8058,14 @@ interface NSString2 : NSSecureCoding, NSMutableCopying, CKRecordValue
 		[Export ("paragraphRangeForRange:")]
 		NSRange GetParagraphRange (NSRange range);
 
+		[Watch (2,0), TV (9,0), Mac (10,0), iOS (2,0)]
+		[Export("componentsSeparatedByString:")]
+		string[] ComponentsSeparatedByString(string separator);
+
+		[Watch (2,0), TV (9,0), Mac (10,5), iOS (2,0)]
+		[Export("componentsSeparatedByCharactersInSet:")]
+		string[] ComponentsSeparatedByCharactersInSet(NSCharacterSet separator);
```

Comment:
ComponentsSeparatedByCharactersInSet -> ComponentsSeparatedByCharacters

Reply:
same, the name lack an action/veb